### ISSUE TITLE
(vue) TSLint is running on empty script tags

### ISSUE THREAD

**Issue by Toilal** [TypeStrong/fork-ts-checker-webpack-plugin#85]
TSLint is running on empty script tag content, even when a src attribute is defined.

```vue
<template src="./App.html"></template>
<style scoped src="./App.css"></style>
<script lang="ts" src="./App.ts"></script>
```

This raises tslint errors for missing EOL

```
ERROR in C:/devel/projects/vue-ts-test/src/App.vue
(3,91): file should end with a newline
```

**Comment by The-DL** [comment]
I'm seeing this as well. If you move the closing `</script>` to the next line, it doesn't trigger the issue.

**Comment by Toilal** [comment]
@BigForNothing It seems to be fixed with TSLint 5.9.1, can you confirm ?

**Comment by mildful** [comment]
I'm running tslint 5.10.0 and still have this issue. I can also confirm that moving the closing `</script>` tag to the next slide fix this.
